servercfg/serverconf.go

```diff
@@ -427,7 +428,16 @@ func GetPublicIP() (string, error) {
 	var err error
 
 	iplist := []string{"https://ip.server.gravitl.com", "https://ifconfig.me", "https://api.ipify.org", "https://ipinfo.io/ip"}
+	publicIpService := os.Getenv("PUBLIC_IP_SERVICE")
+	if publicIpService != "" {
+		fmt.Println("User provided public IP service is", publicIpService)
```

Comment:
Use logger.Log()

Reply:
Thanks Matt, would like to.  We're working on how to get that done.